src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/PublicationSaveRequest.cs

```diff
@@ -23,4 +24,14 @@ public string Slug
     }
 
     public Guid? SupersededById { get; set; }
+
+     public class Validator : AbstractValidator<PublicationSaveRequest>
+    {
+        public Validator()
+        {
+            RuleFor(request => request.Title)
+                .NotEmpty()
+                .MaximumLength(65);
+        }
+    }
```

Comment:
Same comment as `PublicationCreateRequest.cs`. We are also validating summary is less than 160 chars so we can move this to the Fluent Validation validator to be consistent.

We can also write unit tests for this.

Reply:
Fixed in 4701db8.